play/PlayerRules/Basic Rules.txt

```diff
@@ -131,15 +131,27 @@ Feats		- Your character will gain special perks as you level up. This
 Player Stats:
 ==============================
 
-Each player has what are called Base Stats. An average person has a '5'
-for each stat, but an average player has a '6' . These stats are:
+Each player has what are called Base Stats. An average person has a '5' for
+each stat, but an average player has a '6'. Base Stats are split into Primary
+and Secondary stats. Primary stats are determined by your Base Stat points.
+Secondary stats are derived from several primary stats. For every point in a primary
+stat, a secondary stat point is awarded (if the primary and secondary stats are
+associated). The maximum number of points that can be put into a given secondary stat
+is determined by the quantity of points in the largest associated primary stat.
+These stats are as follows (secondary stat associations are indented):

```

Comment:
The increased player choice, mainly.
1. If carry ability is simply added as a stat, it's worthless compared to getting pure dex for movement speed. Dex gets so much more, up until you hit the point cap. Then all that carry ability offers is cancellation of movements penalties so if you have no movement penalties, it's utterly worthless. I reckoned that having a different way to get the stat seemed prudent. 
2. Instead of forcing a player to pick 10 dex to increase movement, he can pick 10 dex or 10 str. Adding carry ability also gives STR, and especially fortitude, more value. The way it's previously been in CX, if you want to be good at melee you almost always have to get really good dex, because of movement speed. Then if you want a power fist to do damage, you need to have strength, so there is a bunch of your points decided for you by the game. Having a secondary stat system allows players to have more creativity and still be viable. 
3. Charisma has low value besides the extra feat bonus (which is arguably the only reason to take charisma because you can supplement charisma with speech skill, etc. but that's a discussion for another time). Skill points are such a fundamental part of the game, you _have_ to put lots of points into intelligence if you want to be good at RP or reloading your weapon, etc. The secondary stat system allows you to pick charisma or intelligence to increase your skill point gain.

As usual, i'm open to ideas and I'm trying to get the most elegant and useful system built, that I can.